modules/govuk/templates/ckan/pycsw.cfg.erb

```diff
@@ -0,0 +1,61 @@
+[server]
+home=/var/ckan
+url=http://localhost:<%= @pycsw_port %>/csw
+mimetype=application/xml; charset=UTF-8
+encoding=UTF-8
+language=en-GB
+maxrecords=10
+loglevel=INFO
+logfile=/var/log/ckan/pycsw.log
+#ogc_schemas_base=http://foo
+#federatedcatalogues=http://catalog.data.gov/csw
+#pretty_print=true
+gzip_compresslevel=9
+#domainquerytype=range
+#domaincounts=true
+#spatial_ranking=true
+profiles=apiso
+
+[manager]
+transactions=false
+allowed_ips=127.0.0.1
+#csw_harvest_pagesize=10
+
+[metadata:main]
+identification_title=pycsw Geospatial Catalogue
+identification_abstract=pycsw is an OGC CSW server implementation written in Python
+identification_keywords=catalogue,discovery,metadata
+identification_keywords_type=theme
+identification_fees=None
+identification_accessconstraints=None
+provider_name=Organization Name
+provider_url=http://pycsw.org/
+contact_name=Lastname, Firstname
+contact_position=Position Title
+contact_address=Mailing Address
+contact_city=City
+contact_stateorprovince=Administrative Area
+contact_postalcode=Zip or Postal Code
+contact_country=Country
+contact_phone=+xx-xxx-xxx-xxxx
+contact_fax=+xx-xxx-xxx-xxxx
+contact_email=you@example.org
+contact_url=Contact URL
+contact_hours=Hours of Service
+contact_instructions=During hours of service.  Off on weekends.
+contact_role=pointOfContact
+
+[repository]
+database=<%= "postgresql://#{@db_username}:#{@db_password}@#{@db_hostname}/#{@db_name}"%>
+table=records
+
+[metadata:inspire]
+enabled=true
+languages_supported=eng,gre
+default_language=eng
+date=YYYY-MM-DD
+gemet_keywords=Utility and governmental services
+conformity_service=notEvaluated
+contact_name=Government Digital Service
+contact_email=Email Address
```

Comment:
Maybe this should be the Zendesk email address? Assuming it's a regular "contact for help" thing.

Reply:
Is the Zendesk email address exposed anywhere? I think usually we'd want people to go via https://data.gov.uk/support.

Reply:
I'm unsure, regardless, I don't think it matters too much at this stage. Tweaking the contact parameters can be done later if it's useful to do.